src/main/resources/api/service_openapi.yaml

```diff
@@ -540,6 +593,18 @@ components:
       type: string
       enum: ['RAWLS_WORKSPACE', 'MC_WORKSPACE']
 
+    CreateGoogleContextRequestBody:
+      description: Request body object for creating the Google context.
+      type: object
+      required: [name, cloningInstructions]
```

Comment:
this doesn't have name or cloningInstructions fields

Reply:
oops